koku/api/report/test/aws/test_queries.py

```diff
@@ -2658,6 +2658,19 @@ def test_filter_and_group_by_org_unit_match(self):
                         with self.subTest(breakdown):
                             self.assertAlmostEqual(group_dict[breakdown]["value"], filter_dict[breakdown]["value"], 6)
 
+    @patch("api.query_params.enable_negative_filtering", return_value=True)
+    def test_group_by_and_filter_with_excluded_org_unit(self, _):
```

Comment:
If we don't need to modify the `Mock` it might be better to use it as a context manager within the test method to avoid using a throwaway variable name, or give this parameter a descriptive name. Descriptive names make it easier to keep track of multiple patch objects if more are added in the future. It's also clearer to the reader what this parameter is supposed to be without have to infer.